jerry-core/ecma/operations/ecma-objects-general.c

```diff
@@ -246,6 +246,25 @@ ecma_op_general_object_get_property (ecma_object_t *obj_p, /**< the object */
 } /* ecma_op_general_object_get_property */
 
 /**
+ * Checks whether the property name is "length".
+ *
+ * @return true if the property name is length
+ *         false otherwise
+ */
+static bool
+ecma_op_general_object_property_name_is_length (ecma_string_t *property_name_p) /**< property name */
```

Comment:
I don't get this. Our code base is full of `inline`s and `__attr_always_inline___`s to make sure that performance critical paths are inlined. This patch seems to be focusing on performance. Why not force inlining this time then? Especially for a function which is only used once (I do realize that debug version uses it twice but debug mode shouldn't count when optimizing performance), thus inlining does not only help performance but code size as well (as it makes sure that function call, parameter and return value passing, prolog and epilog don't get generated.